src/api/app/controllers/webui2/package_controller.rb

```diff
@@ -1,6 +1,11 @@
 module Webui2::PackageController
   def webui2_show
+    @tprj, @tpkg = project_and_package_names(@package)
```

Comment:
Please do not use these awful variable names.

Reply:
I have some improvements here, but I wanted to fix the tests before introducing more changes